gridsome/lib/app/loadConfig.js

```diff
@@ -113,6 +113,8 @@ module.exports = (context, options = {}) => {
 
   config.images = { ...localConfig.images }
 
+  config.images.defaultBlur = config.images.defaultBlur || 20
```

Comment:
Maybe the default blur should be `40` since that is what it used to be?

Reply:
@tomtev you have any thought on this? (Based my change on the original request in #213  that 40 is too high for the default).

I don't mind having to change back to 40 if agree.

Reply:
You're right, it should be 20 by default :) I'm only concerned about changing the default value in a minor version. But it shouldn't matter too much anyway, or what do you think @tomtev?

Reply:
@hjvedvik Yeah, that makes sense. I've put it back to `40` to keep less surprises in a minor version. Is easy to change again for next version.